code/modules/economy/commodity.dm

```diff
@@ -2900,3 +2880,1072 @@
+	comtype = /obj/item/yoyo
+	desc_buy = "They're OUT OF PRODUCTION and HIGH IN VALUE!"
+	price = 16100
+	baseprice = 16100
+	upperfluc = 7100
+	lowerfluc = -7100
+/datum/commodity/blackmarket/wrestler/rarefig
+	comname = "Rare Plastic Figurine"
+	comtype = /obj/item/toy/figure/rare
+	desc_buy = "If you get any RARE PLASTIC FIGURINES, I want em!"
+	price = 1600
+	baseprice = 1600
+	upperfluc = 800
+	lowerfluc = -800
+/datum/commodity/blackmarket/wrestler/tsunami
+	comname = "Tsunami Spray Bottle"
+	comtype = /obj/item/spraybottle/cleaner/tsunami
+	desc_buy = "The ring can get SWEATY! Sometimes there's BLOOD!"
+	price = 7200
+	baseprice = 7200
+	upperfluc = 400
+	lowerfluc = -400
+/datum/commodity/blackmarket/wrestler/pen
+	comname = "Fancy Pens"
+	comtype = /obj/item/pen/fancy
+	desc_buy = "I'll pay top price for any of these you bring in!"
+	price = 350
+	baseprice = 350
+	upperfluc = 50
+	lowerfluc = -50
+/datum/commodity/blackmarket/wrestler/chompskey
+	comname = "Chomp Key"
+	comtype = /obj/item/device/key/chompskey
+	desc_buy = "I need one of these URGENTLY!"
+	price = 20000
+	baseprice = 20000
+	upperfluc = 2000
+	lowerfluc = -2000
+/datum/commodity/blackmarket/wrestler/hairballkey
+	comname = "Hairball Key"
+	comtype = /obj/item/device/key/hairball
+	desc_buy = "DON'T even ASK!"
+	price = 60100
+	baseprice = 60100
+	upperfluc = 25000
+	lowerfluc = -25000
+/datum/commodity/blackmarket/wrestler/brick
+	comname = "Ceramic Bricks"
+	comtype = /obj/item/brick
+	desc_buy = "I go through A LOT of these in my shows!"
+	price = 100
+	baseprice = 40
+	upperfluc = 40
+	lowerfluc = -40
+/datum/commodity/blackmarket/wrestler/ghostgun
+	comname = "Ectoplasmic Destabilizer"
+	comtype = /obj/item/gun/energy/ghost
+	desc_buy = "Ghosts are real and THEY'RE COMING FOR MY ASS! I need some ASS KICKING GHOST MULCHING LASER GUNS!"
+	price = 10200
+	baseprice = 10200
+	upperfluc = 10000
+	lowerfluc = -10000
+/datum/commodity/blackmarket/wrestler/goldfishcracker
+	comname = "Goldfish Cracker"
+	comtype = /obj/item/reagent_containers/food/snacks/goldfish_cracker
+	desc_buy = "These things are LIFESAVERS when it comes to building muscle! I NEED AS MANY AS YOU CAN GET!"
+	price = 900
+	baseprice = 900
+	upperfluc = 490
+	lowerfluc = -490
+/datum/commodity/blackmarket/wrestler/teapot
+	comname = "Handmade Tetsubin"
+	comtype = /obj/item/reagent_containers/food/drinks/teapot
+	desc_buy = "I'm an avid collector of fancy teapots, and I'd like to get my hand on any interesting ones you find."
+	price = 1400
+	upperfluc = 3000
+	lowerfluc = -3000
+/datum/commodity/blackmarket/wrestler/nassa
+	comname = "NASSA space suit"
+	comtype = /obj/item/clothing/under/gimmick/blackstronaut
+	desc_buy = "I'd LOVE to get my hands on a couple sets of that prototype jumpsuit. WRESTLING IN THE COLD VACCUM OF SPACE! I'll pay a BUNCH for any you find in good condition!"
+	price = 6200
+	upperfluc = 800
+	lowerfluc = -800
+/datum/commodity/blackmarket/wrestler/uplink_telecrystal
```

Comment:
probably should have some way to be named after the current traitor buck of the round.

Also kinda useless considering traitors likely arent going to want to sell their hard earned antag dollarodoos for something they can easily get from just stealing the budget